twilio-auth-sample/src/main/java/com/twilio/authsample/main/TokensFragment.java

```diff
@@ -0,0 +1,70 @@
+package com.twilio.authsample.main;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.twilio.authenticator.TwilioAuthenticator;
+import com.twilio.authenticator.internal.models.data.App;
+import com.twilio.authsample.R;
+import com.twilio.authsample.totp.TokensAdapter;
+
+import java.util.List;
+
+/**
+ * Created by lvidal on 10/12/17.
+ */
+
+public class TokensFragment extends Fragment implements TokensAdapter.OnClickListener {
+    private RecyclerView mRecyclerView;
```

Comment:
I don't mind using Hungarian notation for fields, but since we are not using them on the rest of the app I suggest you remove them. Also it is not used on other new classes from this PR, TokensAdapter for instance